src/app/public/modules/phone-field/phone-field.component.html

```diff
@@ -0,0 +1,45 @@
+<div
+  class="sky-form-group sky-input-group"
+>
+  <div
+    class="sky-input-group-btn sky-input-group-country-btn"
+  >
+    <sky-dropdown
+      [disabled]="disabled"
+      [label]="'skyux_phone_field_dropdown_label' | skyLibResources"
+      [title]="selectedCountry.name + ' +' + selectedCountry.dialCode"
+    >
+      <sky-dropdown-button>
+        <div
+          [className]="'iti-flag ' + selectedCountry.iso2"
+        >
+        </div>
+      </sky-dropdown-button>
+      <sky-dropdown-menu>
+        <sky-dropdown-item *ngFor="let country of countryData; let i = index"
+          [ngClass]="{
+            'sky-selected-country': i === 0,
```

Comment:
These class names are a bit generic; maybe prefix them with the name of the component to "secure" them under the component's namespace?

`sky-phone-field-country-selected`, `sky-phone-field-country-default` (or something similar)?

Reply:
Done